lib/ruby_smb/client/negotiation.rb

```diff
@@ -57,7 +57,6 @@ def negotiate_request
 
       # Takes the raw response data from the server and tries
       # parse it into a valid Response packet object.
-      # This method currently assumes that all SMB1 will use Extended Security.
       #
       # @param raw_data [String] the raw binary response from the server
       # @return [RubySMB::SMB1::Packet::NegotiateResponseExtended] when the response is an SMB1 Extended Security Negotiate Response Packet
```

Comment:
I think we're missing another return comment here now

Reply:
👍